assets/js/modules/analytics/datastore/report.js

```diff
@@ -57,20 +57,23 @@ const fetchGetReportStore = createFetchStore( {
 		invariant( isPlainObject( options ), 'Options for Analytics report must be an object.' );
 		invariant( isValidDateRange( options ), 'Either date range or start/end dates must be provided for Analytics report.' );
 
-		const { metrics, dimensions, orderby } = normalizeReportOptions( options );
+		const { metrics, dimensions, dimensionFilters, orderby } = normalizeReportOptions( options );
 
 		invariant( metrics.length, 'Requests must specify at least one metric for an Analytics report.' );
 		invariant(
 			isValidMetrics( metrics ),
 			'Metrics for an Analytics report must be either a string, an array of strings, an object, an array of objects or a mix of strings and objects. If an object is used, it must have "expression" and "alias" properties.',
 		);
 
-		if ( dimensions ) {
-			invariant(
-				isValidDimensions( dimensions ),
-				'Dimensions for an Analytics report must be either a string, an array of strings, an object, an array of objects or a mix of strings and objects. If an object is used, it must have "name" property.',
-			);
-		}
+		invariant(
+			isValidDimensions( dimensions ),
+			'Dimensions for an Analytics report must be either a string, an array of strings, an object, an array of objects or a mix of strings and objects. If an object is used, it must have "name" property.',
+		);
```

Comment:
Why did you remove the if clause here? I think that's an unrelated change.

Reply:
Yes, unrelated, thanks for noticing it.